src/main/java/com/softech/coursesvc/controller/CourseController.java

```diff
@@ -43,21 +44,27 @@ public ResponseEntity<CourseEntity> getCourseById(@PathVariable Long courseId)
         return new ResponseEntity<CourseEntity>(course, HttpStatus.OK);
     }else {
         return new ResponseEntity<>(null, HttpStatus.NOT_FOUND);
```

Comment:
instead of null , need to add appropriate message.

Reply:
you can ignore it ,, your ticket is aboutupdate endpoint validation